packages/messages/.eslintignore

```diff
@@ -0,0 +1,2 @@
+jest.config.js
+node_modules
```

Comment:
Can we add an empty line here at the end of the file?

You can find information of why we add an empty line at the end of each text file in the following link:
[https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file](https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file)